### PR TITLE
Replace print with logging

### PR DESCRIPTION
**Pull request by huchenlei**
This PR enforces ruff lint rule `T201` that bans usage of `print`. All existing usage of `print` in core are replaced with `logging`. All usages in tests/deployment files are annotated with `# noqa`.

### CODE REVIEW

**Review by robinjhuang — APPROVED** [review]

File: comfy/ldm/aura/mmdit.py [hunk 1]
```diff
@@ -381,7 +381,6 @@ def extend_pe(self, init_dim=(16, 16), target_dim=(64, 64)):
         pe_new = pe_as_2d.squeeze(0).permute(1, 2, 0).flatten(0, 1)
         self.positional_encoding.data = pe_new.unsqueeze(0).contiguous()
         self.h_max, self.w_max = target_dim
-        print("PE extended to", target_dim)
```
Comment (webfiltered) [inline comfy/ldm/aura/mmdit.py:384]: Just confirming - this one is a removal another cleanup of a debug print that probably should have been removed earlier?

Comment (huchenlei) [reply comfy/ldm/aura/mmdit.py:384]: Yes. A few prints in the model files are directly removed, as they were probably added for debugging purpose before.

File: comfy/model_base.py [hunk 2]
```diff
@@ -754,7 +754,6 @@ def concat_cond(self, **kwargs):
             mask = torch.ones_like(noise)[:, :1]
 
         mask = torch.mean(mask, dim=1, keepdim=True)
-        print(mask.shape)
```
Comment (huchenlei) [inline comfy/model_base.py:773]: Cleanup debug logs.

File: comfy/cldm/cldm.py [hunk 3]
```diff
@@ -160,7 +160,6 @@ def __init__(
             if isinstance(self.num_classes, int):
                 self.label_emb = nn.Embedding(num_classes, time_embed_dim)
             elif self.num_classes == "continuous":
-                print("setting up linear c_adm embedding layer")
```
Comment (huchenlei) [inline comfy/cldm/cldm.py:163]: Cleanup debug logs.

File: comfy/extra_samplers/uni_pc.py [hunk 4]
```diff
@@ -518,7 +519,6 @@ def multistep_uni_pc_vary_update(self, x, model_prev_list, t_prev_list, t, order
             A_p = C_inv_p
 
         if use_corrector:
-            print('using corrector')
```
Comment (huchenlei) [inline comfy/extra_samplers/uni_pc.py:521]: Cleanup debug logs.

File: comfy/sd1_clip.py [hunk 5]
```diff
@@ -41,8 +41,7 @@ def encode_token_weights(self, token_weight_pairs):
                 to_encode.append(self.gen_empty_tokens(self.special_tokens, max_token_len))
             else:
                 to_encode.append(gen_empty_tokens(self.special_tokens, max_token_len))
-            print(to_encode)
```
Comment (huchenlei) [inline comfy/sd1_clip.py:44]: Cleanup debugging print added in https://github.com/comfyanonymous/ComfyUI/pull/6055.

**Pull request merged**